server/testkit/defunct_examples/enumerating_access.rs

```diff
@@ -4,7 +4,9 @@
 //!
 
 use kanidmd_lib::constants::entries::Attribute;
-use kanidmd_lib::constants::groups::{idm_builtin_admin_groups, idm_builtin_non_admin_groups};
+use kanidmd_lib::migration_data::current::{
```

Comment:
I feel like I'm crazy, but I can't find where the 'current' module is
![image](https://github.com/user-attachments/assets/52c2c8a7-62fb-4b41-8e09-002f1d9a51cb)

Reply:
The example is defunct, the current module doesn't exist. But this example was just too broken in the current work..